spyder/app/mainwindow.py

```diff
@@ -88,77 +83,44 @@
 # Proper high DPI scaling is available in Qt >= 5.6.0. This attibute must
 # be set before creating the application.
-#==============================================================================
+# ============================================================================
 from spyder.config.manager import CONF
 
-if hasattr(Qt, 'AA_EnableHighDpiScaling'):
-    QCoreApplication.setAttribute(Qt.AA_EnableHighDpiScaling,
-                                  CONF.get('main', 'high_dpi_scaling'))
 
-#==============================================================================
+# ============================================================================
 # Get CLI options and set OpenGL backend. This attibute must
 # be set before creating the application. See spyder-ide/spyder#11227
-#==============================================================================
+# ============================================================================
 from spyder.app.utils import set_opengl_implementation
 from spyder.app.cli_options import get_options
 from spyder.config.base import running_under_pytest
 
-# Get CLI options/args and make them available for future use.
-# Ignore args if running tests or Spyder will try and fail to parse pytests's.
-if running_under_pytest():
-    sys_argv = [sys.argv[0]]
-    CLI_OPTIONS, CLI_ARGS = get_options(sys_argv)
-else:
-    CLI_OPTIONS, CLI_ARGS = get_options()
-
-# **** Set OpenGL implementation to use ****
-if CLI_OPTIONS.opengl_implementation:
-    option = CLI_OPTIONS.opengl_implementation
-    set_opengl_implementation(option)
-else:
-    if CONF.get('main', 'opengl') != 'automatic':
-        option = CONF.get('main', 'opengl')
-        set_opengl_implementation(option)
+CLI_OPTIONS = None
+CLI_ARGS = None
 
-#==============================================================================
+# ============================================================================
 # Create our QApplication instance here because it's needed to render the
 # splash screen created below
-#==============================================================================
+# ============================================================================
 from spyder.utils.qthelpers import qapplication
 from spyder.config.base import get_image_path
-MAIN_APP = qapplication()
 
-if PYQT5:
-    APP_ICON = QIcon(get_image_path("spyder.svg"))
-else:
-    APP_ICON = QIcon(get_image_path("spyder.png"))
+MAIN_APP = None
+APP_ICON = None
 
-MAIN_APP.setWindowIcon(APP_ICON)
-
-#==============================================================================
+# ============================================================================
 # Create splash screen out of MainWindow to reduce perceived startup time.
-#==============================================================================
+# ============================================================================
 from spyder.config.base import _, get_image_path, DEV

```

Comment:
Please move these imports to a better place.

Reply:
Ok